<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora Profesional</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="display_output">
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="grid_buttons">
      <!-- Fila 1 -->
      <item row="0" column="0"><widget class="QPushButton" name="btn_sin"><property name="text"><string>sin</string></property></widget></item>
      <item row="0" column="1"><widget class="QPushButton" name="btn_cos"><property name="text"><string>cos</string></property></widget></item>
      <item row="0" column="2"><widget class="QPushButton" name="btn_tan"><property name="text"><string>tan</string></property></widget></item>
      <item row="0" column="3"><widget class="QPushButton" name="btn_ce"><property name="text"><string>CE</string></property><property name="objectName"><string>btn_ce</string></property></widget></item>
      <item row="0" column="4"><widget class="QPushButton" name="btn_c"><property name="text"><string>C</string></property><property name="objectName"><string>btn_c</string></property></widget></item>
      <!-- Fila 2 -->
      <item row="1" column="0"><widget class="QPushButton" name="btn_sqrt"><property name="text"><string>sqrt(</string></property></widget></item>
      <item row="1" column="1"><widget class="QPushButton" name="btn_ln"><property name="text"><string>ln(</string></property></widget></item>
      <item row="1" column="2"><widget class="QPushButton" name="btn_log"><property name="text"><string>log(</string></property></widget></item>
      <item row="1" column="3"><widget class="QPushButton" name="btn_pow"><property name="text"><string>^</string></property></widget></item>
      <item row="1" column="4"><widget class="QPushButton" name="btn_del"><property name="text"><string>DEL</string></property><property name="objectName"><string>btn_del</string></property></widget></item>
      <!-- Fila 3 -->
      <item row="2" column="0"><widget class="QPushButton" name="btn_7"><property name="text"><string>7</string></property></widget></item>
      <item row="2" column="1"><widget class="QPushButton" name="btn_8"><property name="text"><string>8</string></property></widget></item>
      <item row="2" column="2"><widget class="QPushButton" name="btn_9"><property name="text"><string>9</string></property></widget></item>
      <item row="2" column="3"><widget class="QPushButton" name="btn_div"><property name="text"><string>/</string></property></widget></item>
      <item row="2" column="4"><widget class="QPushButton" name="btn_pi"><property name="text"><string>pi</string></property></widget></item>
      <!-- Fila 4 -->
      <item row="3" column="0"><widget class="QPushButton" name="btn_4"><property name="text"><string>4</string></property></widget></item>
      <item row="3" column="1"><widget class="QPushButton" name="btn_5"><property name="text"><string>5</string></property></widget></item>
      <item row="3" column="2"><widget class="QPushButton" name="btn_6"><property name="text"><string>6</string></property></widget></item>
      <item row="3" column="3"><widget class="QPushButton" name="btn_mul"><property name="text"><string>*</string></property></widget></item>
      <item row="3" column="4"><widget class="QPushButton" name="btn_e"><property name="text"><string>e</string></property></widget></item>
      <!-- Fila 5 -->
      <item row="4" column="0"><widget class="QPushButton" name="btn_1"><property name="text"><string>1</string></property></widget></item>
      <item row="4" column="1"><widget class="QPushButton" name="btn_2"><property name="text"><string>2</string></property></widget></item>
      <item row="4" column="2"><widget class="QPushButton" name="btn_3"><property name="text"><string>3</string></property></widget></item>
      <item row="4" column="3"><widget class="QPushButton" name="btn_sub"><property name="text"><string>-</string></property></widget></item>
      <item row="4" column="4"><widget class="QPushButton" name="btn_paren_l"><property name="text"><string>(</string></property></widget></item>
      <!-- Fila 6 -->
      <item row="5" column="0"><widget class="QPushButton" name="btn_0"><property name="text"><string>0</string></property></widget></item>
      <item row="5" column="1"><widget class="QPushButton" name="btn_dot"><property name="text"><string>.</string></property></widget></item>
      <item row="5" column="2"><widget class="QPushButton" name="btn_pm"><property name="text"><string>+/-</string></property><property name="objectName"><string>btn_pm</string></property></widget></item>
      <item row="5" column="3"><widget class="QPushButton" name="btn_add"><property name="text"><string>+</string></property></widget></item>
      <item row="5" column="4"><widget class="QPushButton" name="btn_paren_r"><property name="text"><string>)</string></property></widget></item>
      <!-- Fila 7 -->
      <item row="6" column="0" colspan="5"><widget class="QPushButton" name="btn_equals"><property name="text"><string>=</string></property><property name="objectName"><string>btn_equals</string></property></widget></item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>